source/_integrations/sensor.rest.markdown

```diff
@@ -29,7 +29,7 @@ sensor:
   - platform: rest
     resource: http://IP_ADDRESS/ENDPOINT
     method: POST
-    payload: "{ "device" : "heater" }"
+    payload: '{ "device" : "heater" }'
```

Comment:
I am not sure about this one; The JSON wants double quotes.

Reply:
Correct, this is an exceptional case :)